### PR TITLE
Fix translations

### PR DESCRIPTION
**Pull request by oterral**
This PR replace uses of `gettextCatalog` by `gettext` to fix the translation and also remove use of tranlsate filter when it's unneeded.

### CODE REVIEW

File: src/modules/import/partials/import-local.html [hunk 1]
```diff
@@ -13,7 +13,8 @@
   <button type="button" class="btn btn-default btn-sm ngeo-load"
           ng-class="{'btn-danger': loading}"
           ng-disabled="!isValid(file) || (loading && !fileReader)"
-          ng-click="loading ? cancel() : handleFile()">
-    <span>{{userMessage | translate }}</span>
+          ng-click="loading ? cancel() : handleFile()"
+          translate>
```
Comment (gberaudo) [inline src/modules/import/partials/import-local.html]: You need to keep using the `translate` filter. Why is this change needed?

File: src/modules/import/partials/import-online.html [hunk 2]
```diff
@@ -8,7 +8,8 @@
   <button type="button" class="btn btn-default btn-sm"
           ng-class="{'btn-danger': canceler}"
           ng-disabled="!fileUrl || !isValid(fileUrl) || (loading && !canceler)"
-          ng-click="loading ? cancel() : handleFileUrl()">
-    {{userMessage | translate}}
+          ng-click="loading ? cancel() : handleFileUrl()"
+          translate>
```
Comment (gberaudo) [inline src/modules/import/partials/import-online.html]: Idem

**Comment by oterral** [comment]
According to angular-gettext doc it should work. But ok if it's necessary.

**Comment by oterral** [comment]
@gberaudo  filter re-added

**Comment by gberaudo** [comment]
> According to angular-gettext doc it should work. But ok if it's necessary.
Really? I tried both and the behaviour were differents.

**Comment by gberaudo** [comment]
Waiting for the build to finish. Will merge as soon as Travis is green.

**Comment by gberaudo** [comment]
Thanks @oterral.
